lms/templates/problem.html

```diff
@@ -42,16 +42,47 @@ <h3 class="hd hd-2 problem-header" id="${ id }-problem-title" aria-describedby="
     </span>
     % endif
     </div>
-    % if check_button:
-    <button class="check ${ check_button } btn-brand" data-checking="${ check_button_checking }" data-value="${ check_button }">
-        <span class="check-label">${ check_button }</span><span class="sr"> ${_("your answer")}</span>
+    % if attempts_allowed:
+    <button type="button" class="submit btn-brand" data-submitting="${ submit_button_submitting }" data-value="${ submit_button }" ${attempts_allowed} data-attempts-remaining="${attempts_allowed - attempts_used}" aria-describedby="submission_feedback_${short_id}" ${'' if should_enable_submit_button else 'disabled'}>
+        <span class="submit-label">${ submit_button }</span><span class="sr"> ${_("your answer")}</span>
+    </button>
+    % else:
+    <button type="button" class="submit btn-brand" data-submitting="${ submit_button_submitting }" data-value="${ submit_button }" aria-describedby="submission_feedback_${short_id}" ${'' if should_enable_submit_button else 'disabled'}>
```

Comment:
The only difference is that data-attempts-remaining is not present (vs. lines 46-47)? If so, break the HTML over multiple lines so the conditional can be put in without duplicating the HTML (since it would be easy to accidentally introduce a bug by only updating one of the 2 cases). For example, 

````
<button type="button" class="submit btn-brand" data-submitting="${ submit_button_submitting }" data-value="${ submit_button }"
% if attempts_allowed:
    data-attempts-remaining="${attempts_allowed - attempts_used}"
% endif
${'' if should_enable_submit_button else 'disabled'}>